pkg/controller/status/status.go

```diff
@@ -106,6 +106,7 @@ type statusManager struct {
 	// if there are any, and report statuses based on the state of those resources.
 	readyToMonitor bool
 	hasSynced      bool
+	crExists       bool
```

Comment:
Should we default this to true to ensure if the operator is restarted that an old status would get cleaned up if needed?

Reply:
Yeah that's probably a good idea.

Reply:
Maybe a better idea would be to query for the CR in the New call so we initialize to the real value.